src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/EnvelopeParser.java

```diff
@@ -19,7 +19,7 @@
     public static Envelope parse(byte[] bytes) {
         try {
             return objectMapper.readValue(bytes, Envelope.class);
-        } catch (IOException exc) {
+        } catch (IOException | NullPointerException exc) {
```

Comment:
not sure we should be doing this, catching null pointer is an anti pattern.
either add a null check first or fix the code that is causing null to be sent?